### ISSUE TITLE
New add-on submissions form broken

### ISSUE THREAD

**Issue by hhyyrylainen** [mozilla/addons-server#4254]
I'm trying to get my first addon (a webextension) signed so I could permanently install it. But I cannot write in the name or summary fields on the new addon submission form and thus I cannot submit it.

I first thought that maybe it was the localized names that caused the issue but I tried also with a plain text name and the issue was the same. Using the website inspector it seems that the Name and Description fields are missing the class (slug-edit) that adds the edit box thing. Whereas the add-on url field has it and it works correctly.

Here's a screenshot of me trying to submit an addon with a simple name and description and it still isn't working (I did try changing language and it was still the same): 

![screenshot showing the issue](http://i.imgur.com/ixmesnC.png])

Also I cannot delete the old attempts in order to get a sensible id for my addon. It uses a native extension so changing the id means I have to update the native extension allowed addons, too.
Can I or someone else delete them so I could re-use their ids or do I have to put uuids in my addon ids?

**Comment by EnTeQuAk** [comment]
I think @wagnerand or @jvillalobos can help deleting things so you may re-use the ids.

For the issue, this is interesting to see. As a workaround, does this work if you're not using the Finish locale but English (+ maybe none localized name/summary)?

Have you set `default_locale` for your add-on? If not we'll use your user locale which is Finish and might cause issues here. 

We'll take a look at this and see what's wrong. Could be related to #3899 and the user-locale being used by default, we might need to add some warnings / errors to fix this. Not sure yet.

**Comment by wagnerand** [comment]
I don't think deletion is necessary here. You can resume the submission process from your developer hub.

**Comment by hhyyrylainen** [comment]
@wagnerand I switched to the English version of the site and now I can see my submission attempts. And I can try to resume them. But I found out that when I have a localized name for a plugin it doesn't show up on "Manage My Submissions"  unless the language is English, which is pretty weird. 

@EnTeQuAk Even when I switch the site language to English (US) I'm still not able to edit the name.

I have this is my manifest.json:
```
    "manifest_version": 2,
    "name": "__MSG_extensionName__",
    "description": "__MSG_extensionDescription__",
    "version": "1.0",
    "default_locale": "en",

    scripts etc. here...
```

Alright I tested with a plain text name and it works even when I switch back to Finnish. I have no idea why it works now but didn't work before when I tried a non-localized name.

I was now able to submit my addon by copying the actual name from the localization files to the manifest. Though it would be nice if this was fixed because I was following the webextension tutorials on mozilla.org and it is pretty bad that the documentation shows what NOT to do.
